src/test/java/uk/gov/hmcts/cmc/claimstore/utils/ResourceLoader.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.cmc.claimstore.utils;
+
+import uk.gov.hmcts.cmc.claimstore.processors.JsonMapper;
+import uk.gov.hmcts.cmc.claimstore.repositories.mapping.JsonMapperFactory;
+import uk.gov.hmcts.cmc.domain.utils.ResourceReader;
+import uk.gov.hmcts.document.domain.UploadResponse;
+
+public class ResourceLoader {
+    private static final JsonMapper jsonMapper = JsonMapperFactory.create();
+
+    private ResourceLoader() {
+    }
+
+    public static UploadResponse documentManagementUploadResponse() {
+        final String response = new ResourceReader().read("/document-management-response.json");
+        return jsonMapper.fromJson(response, UploadResponse.class);
+    }
+
+
+    public static UploadResponse failedDocumentManagementUploadResponse() {
+        final String response = "{\n"
```

Comment:
I would extract this to a JSON file for readability. Even for such a small document it looks now so good. We could have `/document-management-response.failure.json` The other one could get renamed to be `/document-management-response.success.json` for consistency.

Reply:
done